bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Proposal.scala

```diff
@@ -74,12 +76,16 @@ object Proposal {
     List.empty[Keyword],
     Investigators.empty,
     List.empty[Target],
-    List.empty[Observation],
+    List(Observation.empty),
```

Comment:
Now instead of an empty `List[Observation]`, we initialize a new `Proposal` with a `List[Observation]` with an empty observation.